src/anchoredinterval.jl

```diff
@@ -160,12 +186,25 @@ function Base.convert(::Type{AnchoredInterval{P}}, interval::Interval{T}) where
 end
 =#
 
-function Base.convert(::Type{AnchoredInterval{Ending}}, interval::Interval{T,L,R}) where {T,L,R}
-    AnchoredInterval{-span(interval), T, L, R}(last(interval))
+one(T::Type{<:TimeType}) = eps(T)
```

Comment:
this seems highly confusing.

If i see `one` in code, i expect it to be `Base.one` (or have the same semantics) which is to be the multiplicative identity (which `eps(T)` is not)

Reply:
I renamed this to `_span_fallback`

Reply:
thanks!